zeq/detail/event.h

```diff
@@ -25,13 +23,7 @@ class Event
 public:
     Event( const uint128_t& type_ )
         : type( type_ )
-    {
-        const std::string& schema = vocabulary::detail::getSchema( type );
-
-        // populate parser with schema from type
-        if( !schema.empty() && !parser.Parse( schema.c_str( )))
-            LBTHROW( std::runtime_error( parser.error_ ));
-    }
+    {}
```

Comment:
that change in behavior will affect RESTBridge if I'm not mistaken. Why the change?